doc/adr006-modules.lit.md

```diff
@@ -556,24 +559,36 @@ Using this syntax, our example looks like follows:
 
 ```bluespec
 module fixed {
-  include general(N = 3) as I3
-  include general(N = 5) as I5
-  
+  import general(N = 3).*
+  import general(N = 5) as I5
```

Comment:
I think we should also support normal imports with qualifiers like this.
```suggestion
  import general(N = 5) as I5
  import statelessModule as M
```

Reply:
> I think we should also support normal imports with qualifiers like this.

Totally. There is no big conceptual difference.